src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action=" " 
+      method="get"
+      data-qa="big"
+      class="bigSearch"
+      >
+      <img 
+      src="./images/Search.svg" 
+      alt="bigImg"
+      class="bigImg"
+      >
+      <input 
+      type="text" 
+      name="search" 
+      id="big_search" 
+      class="big_search search"
+      placeholder="Try &quot;Los Angeles&quot;"
+      data-qa="keypress"
+      >
```

Comment:
```suggestion
    <form 
        action=" " 
        method="get"
        data-qa="big"
        class="bigSearch"
      >
      <img 
        src="./images/Search.svg" 
        alt="bigImg"
        class="bigImg"
      >
        <input 
        type="text" 
        name="search" 
        id="big_search" 
        class="big_search search"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
```

Reply:
same for another form  https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md